src/components/YextEntityFieldSelector.tsx

```diff
@@ -87,8 +98,37 @@ export const YextEntityFieldSelector = <T extends Record<string, any>>(
               />
             </FieldLabel>
           )}
+          {shouldDisplayConstantValueField(props.filter.types) && (
+            <ToggleMode
+              entityFieldMode={entityFieldMode}
+              toggleEntityFieldMode={toggleEntityFieldMode}
+            />
+          )}
         </>
       );
     },
   };
 };
+
+const ToggleMode = ({
+  entityFieldMode,
+  toggleEntityFieldMode,
+}: {
+  entityFieldMode: boolean;
+  toggleEntityFieldMode: () => void;
+}) => {
+  return (
+    <div className="ve-mt-2">
```

Comment:
![Screenshot 2024-10-02 at 2 33 18 PM](https://github.com/user-attachments/assets/2e15ee51-efc6-4ee3-82ec-ec54125ed15e)
I think this needs some additional layout styles